accounting/urls.py

```diff
@@ -22,22 +22,31 @@
 """
 
 from django.conf import settings
-from django.conf.urls import include, url
+from django.conf.urls import include, static, url
 from django.contrib import admin
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^account/', include('accounting.account.urls', namespace='account')),
+    url(r'^address/', include('accounting.address.urls', namespace='address')),
     url(r'^auth/', include('accounting.authentication.urls', namespace='auth')),
     url(r'^bank/', include('accounting.bank.urls', namespace='bank')),
     url(r'^invoice/', include('accounting.invoice.urls', namespace='invoice')),
     url(r'^registration/', include('accounting.registration.urls', namespace='registration')),
 ]
 
-if settings.DEBUG and settings.ENABLE_DEBUG_TOOLBAR:
-    import debug_toolbar
-    # "debug" URL pattern must be before "site" URL pattern.
-    # See https://github.com/jazzband/django-debug-toolbar/issues/529
+if 'rosetta' in settings.INSTALLED_APPS:
```

Comment:
@UmanShahzad Add a comment to mention what we are using rosetta URLs for, perhaps?

Reply:
I'm contemplating removing this app. I had some initial ideas of how to use it but it didn't manifest. So yeah, just gonna remove it.